### ISSUE TITLE
[GOAL] Make "What is IOTA?" understandable by the layman

### ISSUE THREAD

**Issue by antonionardella** [iota-community/X-Team_IOTA_Simplify#3]
# Make "What is IOTA?" understandable by the layman
**Priority: (B)**

## Key results
1. - [ ] 3 ELI5 infographics
1. - [ ] 3 ELI5 videos
1. - [ ] Revisited IOTA Glossary
2. - [ ] Collection of materials added to the _Awesome IOTA_ repo https://github.com/iota-community/awesome-iota 

# Team
IOTA Simplify - IOTA Experience Team

### IOTA Foundation member

# Additional Information
* Awesome IOTA Repository: https://github.com/iota-community/awesome-iota

# Governance
Please use the following emoji reactions in this post to indicate:
:+1: - Participating
:heavy_plus_sign: - Add my vote

**Comment by Phyloiota** [comment]
We think it will be useful to have some slides on hand to compare the pro's and contras of IOTA to some other projects. Reliable Data metrics need to be collected first. This should be verifiable data - not guessing or heresay. Later on this can be used to create slides, tables, graphics, diagrams etc that will be easy to understand. Data collect sheet can be found here: https://docs.google.com/spreadsheets/d/14hHQskOXTyRA64im77Vjr8PRvtXCU3BeplwFI4F_3rE/edit?usp=sharing
